build.gradle

```diff
@@ -14,9 +14,9 @@ ext.mainClassName = "com.datastax.sparkstress.SparkCassandraStress"
 
 def determineConnectorVersion() {
     if (against == 'dse') {
-        def connector = fileTree(dir: "$DseResources/spark/lib", include: '*connector_*.jar')
+        def connector = fileTree(dir: "$DseResources/spark/lib", include: 'spark*connector*_*.jar')
         def connectorJarName = (connector as List)[0].name
-        def match = connectorJarName =~ /connector_.*-(\d+\.\d+.\d+).*\.jar/
+        def match = connectorJarName =~ /connector.*_.*-(\d+\.\d+.\d+).*\.jar/
```

Comment:
Would `connector.*_.*` have issues with greediness? Typically I do `.*?` or something like `[^_]*` to be cautious.
+1 if it's been tested.